From 8777609aa7fa408228098814e6191f7d4f3d29d2 Mon Sep 17 00:00:00 2001
From: Quihico <Quihico@gmail.com>
Date: Fri, 27 Nov 2015 08:37:10 +0100
Subject: [PATCH 08/16] Added registration script start

Removed Connectivity() and Open_URL() as it is no longer needed.
Start registration-script straigth after wizard.
---
 src/oe.py | 39 +--------------------------------------
 1 file changed, 1 insertion(+), 38 deletions(-)

diff --git a/src/oe.py b/src/oe.py
index 3b6eb33..c1838fb 100644
--- a/src/oe.py
+++ b/src/oe.py
@@ -481,48 +481,11 @@ def openWizard():
         winOeMain = oeWindows.wizard('wizard.xml', __cwd__, 'Default', oeMain=__oe__)
         winOeMain.doModal()
         winOeMain = oeWindows.mainWindow('mainWindow.xml', __cwd__, 'Default', oeMain=__oe__)  # None
+        xbmc.executebuiltin('RunAddon(script.openwindow)')
     except Exception, e:
         xbmc.executebuiltin('Dialog.Close(busydialog)')
         dbg_log('oe::openWizard', 'ERROR: (' + repr(e) + ')')
 
-def Connectivity_Check():
-    internetcheck = 1
-    try:
-        Open_URL('http://google.com')
-    except:
-        try:
-            Open_URL('http://google.com')
-        except:
-            try:
-                Open_URL('http://google.com')
-            except:
-                try:
-                    Open_URL('http://google.cn')
-                except:
-                    try:
-                        Open_URL('http://google.cn')
-                    except:
-                        internetcheck=0
-    if internetcheck==1:
-        xbmc.executebuiltin('RunAddon(script.openwindow)')
-    if internetcheck==0:
-        datapath = xbmc.translatePath(os.path.join('special://profile/addon_data/service.openelec.settings/',''))
-        if os.path.exists(datapath)==True:
-            shutil.rmtree(datapath)
-        try:
-            xbmc.executebuiltin('RunScript(/usr/share/kodi/addons/service.openelec.settings/service.py)')
-        except:
-            xbmc.executebuiltin('RunScript(special://home/addons/service.openelec.settings/service.py)')
-
-
-def Open_URL(url):
-    req = urllib2.Request(url)
-    req.add_header('User-Agent' , 'Mozilla/5.0 (Windows; U; Windows NT 10.0; WOW64; Windows NT 5.1; en-GB; rv:1.9.0.3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36 Gecko/2008092417 Firefox/3.0.3')
-    response = urllib2.urlopen(req)
-    link     = response.read()
-    response.close()
-    return link.replace('\r','').replace('\n','').replace('\t','')
-
 
 def openConfigurationWindow():
     global winOeMain, __cwd__, __oe__, dictModules
-- 
1.9.1

